examples/seismic/accuracy.ipynb

```diff
@@ -0,0 +1,433 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from examples.seismic.acoustic import AcousticWaveSolver\n",
+    "from examples.seismic import Model, RickerSource, Receiver\n",
+    "from devito import clear_cache\n",
+    "from devito.logger import set_log_level\n",
```

Comment:
I *think* you can import this from devito directly

Reply:
Can't nope :)

Reply:
Can you add ``set_log_level`` here:
https://github.com/opesci/devito/blob/master/devito/__init__.py#L8
and try again

Reply:
done